content/tracing/services/_index.md

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
I think the way this section is laid out is still pretty confusing.  The Service List page isn't really the top level here.  Perhaps this would be easier to digest as:

* **Services, Resources, and Traces** (maybe needs a shorter name) - top level page that explains that "APM lets you monitor your application at several levels of granularity - the broadest is services.  A service is...  service level metrics give you an indication of the overall health of your application... a resource is... a trace is..."
  * **Service List** - all of the things you can do/see on the service list page
  * **Service and Resource Dashboards** - all the things you see/can do on the automatic service/resource dashes
  * **Traces** - all the data you see in a [trace view](https://app.datadoghq.com/apm/trace/907546124815982140), also can talk about the [Trace Search page](https://app.datadoghq.com/apm/search)

Reply:
The service List page is the in-app Top level page along with the trace search.
The documentation just reflects the user flow currently two user-flow are available:

1. Service list => service dashboard => resource dashboard => trace => service/resource dashboard
2. Trace search => trace => service/resource dashboard

Do you want to merge all doc regarding the service list in the `/tracing/services/service` page, and keep `/tracing/service` just as a short overview of what's in the section?

Reply:
@l0k0ms I still think this is hard for a new user.  We talk about services / resources etc. before we explain what they are and where they come from.  I also think it's easier to understand if you talk about what all of these components are on the same page: "APM breaks your application down into services, resources, and individual traces..." rather than defining them on separate pages.  It's harder to get the whole picture that way.